raft.go

```diff
@@ -174,44 +189,43 @@ func (r *Raft) runCandidate() {
 	r.logger.Printf("[INFO] raft: %v entering Candidate state in term %v",
 		r, r.getCurrentTerm()+1)
 	metrics.IncrCounter([]string{"raft", "state", "candidate"}, 1)
+	// defer metrics.MeasureSince([]string{"raft", "candidate", "electSelf"}, time.Now())
```

Comment:
I don't actually know where these go, so it's hard for me to tell if I screwed anything up. Do we want this one, and what does it represent?